test/gst-va/encode/av1.py

```diff
@@ -0,0 +1,126 @@
+###
+### Copyright (C) 2019-2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.va.util import *
+from ....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("av1", "encode", "8bit")
+spec_r2r  = load_test_spec("av1", "encode", "8bit", "r2r")
+
+@slash.requires(*have_gst_element("vaav1enc"))
+@slash.requires(*have_gst_element("vaav1dec"))
+class AV1EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "av1-8",
+      gstencoder    = "vaav1enc",
+      gstdecoder    = "vaav1dec",
+      gstmediatype  = "video/x-av1",
+      gstmuxer      = "matroskamux",
+      gstdemuxer    = "matroskademux",
+      gstparser     = "av1parse",
+    )
+
+  def get_file_ext(self):
+    return "webm"
+
+@slash.requires(*platform.have_caps("vdenc", "av1_8"))
+class AV1EncoderLPTest(AV1EncoderBaseTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      caps      = platform.get_caps("vdenc", "av1_8"),
+      # NOTE: vaav1enc does not have lowpower property.
+      # vaav1enc lowpower is hardcoded internally
```

Comment:
Is this really true?  I think gst-msdk is the only one.

Reply:
I'm not sure, because gst-va/encode/av1.py is copied from gst-msdk/encode/av1, and I'm verifying step by step.